e2e/fixtures/fs-router/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "fs-router",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "VITE_EXPERIMENTAL_WAKU_ROUTER=true waku dev",
+    "build": "VITE_EXPERIMENTAL_WAKU_ROUTER=true waku build",
+    "start": "VITE_EXPERIMENTAL_WAKU_ROUTER=true waku start"
```

> I need help with how to correctly set VITE_EXPERIMENTAL_WAKU_ROUTER=true on this PR

You just don't need to set it here.

```suggestion
    "dev": "waku dev",
    "build": "waku build",
    "start": "waku start"
```